.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -2,185 +2,189 @@ name: Bug Report
-      - Firefox
-      - Edge
-      - Safari
-      - Internet Explorer
-      - None (Server)
-      - Other
-  validations:
-    required: true
-    
-- type: input
-  attributes:
-    label: Regression
-    description: "If this behavior worked before, please enter the last working version(s)."
-    placeholder: "@azure/msal-browser 2.0.0"
-    
-- type: dropdown
-  attributes:
-    label: Source
-    description: "Are you an internal or external customer?"
-    options: 
-      - Internal (Microsoft)
-      - External (Customer)
-  validations: 
-     required: true
-    
-- type: markdown
-  attributes:
-    value: "## Security Reporting"
-- type: markdown
-  attributes:
-    value: |
-      If you find a security issue with our libraries or services [please report it to the Microsoft Security Response Center (MSRC)](https://aka.ms/report-security-issue) with as much detail as possible. Your submission may be eligible for a bounty through the [Microsoft Bounty](http://aka.ms/bugbounty) program. Please do not post security issues to GitHub Issues or any other public site. We will contact you shortly upon receiving the information. We encourage you to get notifications of when security incidents occur by visiting [this page](https://www.microsoft.com/msrc/technical-security-notifications) and subscribing to Security Advisory Alerts.
+    - type: markdown
+      attributes:
+          value: |
+              ## Before Submitting your Issue
+              Our GitHub issues are meant exclusively for bug reports and feature requests. Usage questions may be closed without comment. If you have a question, please check to see if your question is answered in the FAQ for the library you are using:
+              - [@azure/msal-browser](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/FAQ.md)
+              - [@azure/msal-node](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/faq.md)
+              - [@azure/msal-angular](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/FAQ.md)
+              - [@azure/msal-react](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/FAQ.md)
+              - [@azure/msal or msal](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-lts/lib/msal-core/docs/FAQ.md)
+
+              In the case your question is not answered in the FAQ, please verify your question has not been covered on [Stack Overflow](https://stackoverflow.com/questions/tagged/msal.js).
+              If you can't find a similar question on Stack Overflow, please create a new Stack Overflow thread using the tag `msal.js`.
+
+              If you have questions about timelines, please see our [roadmap](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/roadmap.md). Issues pertaining to timelines may be closed without comment.
```

Something about how they can still ask for timeline updates on existing feature request issues though
```suggestion
              If you have questions about timelines, please see our [roadmap](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/roadmap.md). Issues pertaining to timelines without an explicit feature request may be closed without comment.
```